index.html

```diff
@@ -42,23 +42,195 @@ <h1>I'm Chrispaix Glad to see you!</h1>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="work">
+            <div class="image">
+                <img src="img/work3.png" alt="My third work">
+            </div>
+            <div class="details">
+                <h3>Tonic</h3>
+                <div class="w-description">
+                    <h4>Canopy</h4>
+                    <ul>
+                        <li>Back End dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="read-more">
+                    <p>
+                        A daily selection of privately personalized reads;
+                         no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="tech-used">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="work">
+            <div class="image">
+                <img src="img/work4.png" alt="My fourth work">
+            </div>
+            <div class="details">
+                <h3>Multi-Post Stories</h3>
+                <div class="w-description">
+                    <h4>Canopy</h4>
+                    <ul>
+                        <li>Back End dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="read-more">
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="tech-used">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button">See Project</button>
                 </div>
             </div>
         </div>
     </section>
 
+    <!--****ABOUT MYSELF****-->
+    <section class="about-me">
+        <div class="container">
+            <div class="aboutme-title">
+                <h2>About Myself</h2>
+            </div>
+            <div class="aboutme-description">
+                <p>
+                    Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <span>Let's connect</span>
```

Comment:
Rather than using `span` for text let's convert it to use `hx` tag except `h1`. Since we only have to have one `h1` on our website and you have it above you can use something from `h2-h6` which every works as the Figma design.